src/test/java/api/support/fixtures/NoticeTokens.java

```diff
@@ -0,0 +1,31 @@
+package api.support.fixtures;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class NoticeTokens {
+
+  private NoticeTokens() {
+  }
+
+  public static final List<String> EXPECTED_USER_TOKENS = Arrays.asList(
+    "user.firstName", "user.lastName", "user.barcode");
+
+
```

Comment:
There is an extra blank line here. Is there a reason for this?

Reply:
No reason for this, removed